.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea/feature for this project
+---
+<!-- **Are you in the right place?**
+1. For issues or feature requests, please create an issue in this repository.
+2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://Rook-io.slack.com).
+3. Did you already search the existing open issues for anything similar? -->
+
+Is this a bug report or feature request?
+* Feature Request
+
+**Feature Request**
+
+Are there any similar features already existing:
+
+What should the feature do:
+
+What would be solved through this feature:
+
+Does this have an impact on existing features:
```

Comment:
How would we expect non-developers to know this?

Reply:
This is just meant when a user knows if there is an impact that he can directly add it.

Reply:
But why would a user know such a thing and not the developers who would implement it? This is something that can't be known until there's a design in place for the implementation. We can't expect anyone to know this at the time of feature request. I think this is an unnecessary field.